src/app/static/src/app/store/plotSelections/plotSelections.ts

```diff
@@ -27,8 +27,18 @@ export const initialPlotSelectionsState = (): PlotSelectionsState => {
     return Object.fromEntries(plotNames.map(plot => [plot, emptySelections])) as PlotSelectionsState;
 }
 
+export const plotSelectionsGetters = {
+    plotControls: (state: PlotSelectionsState) => (plotName: PlotName) => {
+        return state[plotName].controls;
+    },
+    outputFilters: (state: PlotSelectionsState) => (plotName: PlotName) => {
+        return state[plotName].filters;
+    },
+};
```

Comment:
this logic is quite simple can be put into the component itself, i also just realised, while we were discussing I was a little hesitant on getters but i couldnt figure out why so i left it, i now realise why, they are typed as `any`s so you may have to type cast them in the component, i personally just like putting this logic in the computed in the component if it is this simple. I think most getters are like 2 lines ish in this PR, might be worth putting into components